examples/game-server/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "game-server"
+version = "0.1.0"
+edition = "2024"
```

Comment:
Edition 2024 is not currently supported on the MSRV of the crate. Could we drop this?